docs/cpp2/common.md

```diff
@@ -216,11 +216,11 @@ Postfix notation lets the code read fluidly left-to-right, in the same order in
 | `#!cpp --` | `#!cpp iter--` | `#!cpp --iter` |
 | `(` `)` | `#!cpp f( 1, 2, 3)` | `#!cpp f( 1, 2, 3)` |
 | `[` `]` | `#!cpp vec[123]` | `#!cpp vec[123]` |
-| `...` | `#!cpp v.begin()...v.end()` | `#!cpp std::ranges::subrange(v.begin(), v.end())` |
+| `..<` | `#!cpp v.begin()..<v.end()` | `#!cpp std::ranges::subrange(v.begin(), v.end())` |
```

Comment:
@hsutter Should these be in the binary operators table instead of the unary operators table?

Reply:
Good point, I'll look at that post-merge

Reply:
@hsutter 
> Note: The `(` `)`, `[` `]`, `..<`, and `..=` operators are treated as postfix unary operators, though they can take additional arguments.

I can kinda see how `()` and `[]` as pairs are considered postfix since they apply their contents to the thing to the left.  Can you explain how `..<` and `..=` are treated as postfix unary?  I don't understand what that means.